Detector/DetFCChhHCalTile/src/HCalBarrel_geo.cpp

```diff
@@ -93,114 +111,165 @@ static DD4hep::Geometry::Ref_t createHCal(DD4hep::Geometry::LCDD& lcdd, xml_h xm
-    for (unsigned int idxLayer = 0; idxLayer < numSequencesR; ++idxLayer) {
-      auto layerName = DD4hep::XML::_toString(idxLayer, "layer%d");
-      unsigned int sequenceIdx = idxLayer % 2;
-      DetElement layerDet(moduleDet, layerName, idxLayer);
-
-      // in Module rmin = 0  for first wedge, changed radius to the full radius starting at (0,0,0)
-      double rminLayer = sensitiveBarrelRmin + idxLayer * sequenceDimensions.dr();
-      double rmaxLayer = sensitiveBarrelRmin + (idxLayer + 1) * sequenceDimensions.dr();
-      double dx1 = tn * rminLayer - spacing;
-      double dx2 = tn * rmaxLayer - spacing;
-      double rMiddle = (rminLayer - sensitiveBarrelRmin) + 0.5 * sequenceDimensions.dr() - dzModule;
-
-      Volume layerVolume("layerVolume", DD4hep::Geometry::Trapezoid(dx1, dx2, dy0, dy0, dz0), lcdd.material("Air"));
-      layerVolume.setVisAttributes(lcdd.invisible());
-      unsigned int idxSubMod = 0;
-      unsigned int idxActMod = 0;
-      double modCompZOffset = -sequenceDimensions.dz() * 0.5;
-
-      // this matches the order of sequences of standalone HCAL geo description
-      if (sequenceIdx == 0) {
-        sequenceIdx = 1;
-      } else {
-        sequenceIdx = 0;
-      }
-      // Filling of the subWedge with coponents (submodules)
-      for (xml_coll_t xCompColl(sequences[sequenceIdx], _U(module_component)); xCompColl; ++xCompColl, ++idxSubMod) {
-        xml_comp_t xComp = xCompColl;
-        auto compName = DD4hep::XML::_toString(idxLayer, "layer%d") + DD4hep::XML::_toString(idxSubMod, "_tile%d");
-        std::string subModuleName = xComp.materialStr() + DD4hep::XML::_toString(idxSubMod, "_tile%d");
-        DetElement compDet(layerDet, compName, idxSubMod);
-        // added * 0.5 for Trapezoid
-        double dyComp = xComp.thickness() * 0.5;
-        Volume modCompVol("modCompVolume", DD4hep::Geometry::Trapezoid(dx1, dx2, dyComp, dyComp, dz0),
-                          lcdd.material(xComp.materialStr()));
-        modCompVol.setVisAttributes(lcdd, xComp.visStr());
-        DD4hep::Geometry::Position offset(0, modCompZOffset + dyComp + xComp.y_offset() / 2, 0);
-        PlacedVolume placedModCompVol = layerVolume.placeVolume(modCompVol, offset);
-
-        if (xComp.isSensitive()) {
-          modCompVol.setSensitiveDetector(sensDet);
-          placedModCompVol.addPhysVolID("tile", idxActMod);
-          idxActMod++;
-        }
-        compDet.setPlacement(placedModCompVol);
-        modCompZOffset += xComp.thickness() + xComp.y_offset();
-      }
-      DD4hep::Geometry::Position modOffset(0, 0, rMiddle);
-      if ((rMiddle + 0.5 * sequenceDimensions.dr()) > dzModule) {
-        lLog << MSG::WARNING << "something's wrong with the positions in rho!" << endmsg;
+  lLog << MSG::DEBUG << "half width in phi (rmin) of full module (trapazoid side): " << dx1Module << endmsg;
+  lLog << MSG::DEBUG << "half width in phi (rmax) of full module (trapazoid side): " << dx2Module << endmsg;
```

Comment:
Typo: trapazoid -> trapezoid

Isn't this also confusing because you give the width in `cm` not `rad` (or whatever angular measurement)